### PR TITLE
Keep Syntax Highlight Rules Synced With Resources

### LINKED ISSUE #7653: Initial ViewPatch on File
## Current behaviour

image

First time selecting vb file
## Expected behaviour
image

Second time result

## Steps to reproduce

Select a vb file in diff or file history or file list.  Anywhere we show a diff.  First time it fails.  
## Environment

- Git Extensions 3.3.1.7897
- Build 5a97671645532bcedc443bac7b727f40db47cb5c
- Git 2.25.0.windows.1
- Microsoft Windows NT 10.0.18363.0
- .NET Framework 4.8.4075.0
- DPI 96dpi (no scaling)

### PR DESCRIPTION
**Pull request by vbjay**
Fixes #7653

## Proposed changes

- Rename vb highlight rules file
- Add test to keep names and SyntaxNodes in sync
- ~~Switch ICSharp project to debug~~

## Test methodology 

- Renamed an xshd file and ran test
- Added a copy of one of the mode elements and altered the file attribute.

## Test environment(s) 

----

:black_nib: I contribute this code under [The Developer Certificate of Origin](../blob/master/contributors.txt).

### CODE REVIEW

File: GitExtensions.sln [hunk 1]
```diff
@@ -393,8 +393,8 @@ Global
 		{2B9B62FA-3E0C-45C5-9287-68B69413C5DF}.Debug|Any CPU.Build.0 = Debug|Any CPU
 		{2B9B62FA-3E0C-45C5-9287-68B69413C5DF}.Release|Any CPU.ActiveCfg = Release|Any CPU
 		{2B9B62FA-3E0C-45C5-9287-68B69413C5DF}.Release|Any CPU.Build.0 = Release|Any CPU
-		{2D18BE89-D210-49EB-A9DD-2246FBB3DF6D}.Debug|Any CPU.ActiveCfg = Release|Any CPU
-		{2D18BE89-D210-49EB-A9DD-2246FBB3DF6D}.Debug|Any CPU.Build.0 = Release|Any CPU
+		{2D18BE89-D210-49EB-A9DD-2246FBB3DF6D}.Debug|Any CPU.ActiveCfg = Debug|Any CPU
+		{2D18BE89-D210-49EB-A9DD-2246FBB3DF6D}.Debug|Any CPU.Build.0 = Debug|Any CPU
```
Comment (RussKie) [inline GitExtensions.sln]: Please revert, I vaguely remember there was an issue using the Debug build

Comment (vbjay) [reply GitExtensions.sln]: Ok.

**Comment by vbjay** [comment]
Ready to be merged.  Looks like we have some tests that decide to randomly fail.

**Comment by mstv** [comment]
> Looks like we have some tests that decide to randomly fail.

There are several PRs in the pipeline -- two waiting for review. The others depend on these two.

**Comment by RussKie** [comment]
It doesn't look like a correct ICT commit is referenced. I expected this:
image

Other than that, looks good to merge

**Comment by vbjay** [comment]
Updating to point to merge in a sec.

**Comment by vbjay** [comment]
I am having trouble understanding ICT.  Tried looking it up.  I get your request just never heard of ICT.  Mind removing my ignorance?  Thanks.

**Comment by RussKie** [comment]
Apologies for being unclear, I meant "**IC**SharpCode.**T**extEditor"
The current commit referenced in this PR is not pointing to the latest master

**Comment by RussKie** [comment]
Just realised, it is now :) All good

**Pull request merged**